### ISSUE TITLE
Loss function in main_mnist.py

### ISSUE THREAD

**Issue by zhaohui-yang** [itayhubara/BinaryNet.pytorch#11]
Line 86, log softmax
Line 94, Cross Entropy Loss

In the mnist example, you combine Cross Entropy Loss with log softmax, why not using NLLLoss + logsoftmax?

**Comment by alldbi** [comment]
Yes, you are right. It should be either only corss_entropy or NLL+logsoftmax.
